### DISCUSSION TITLE
Alarm Volume Always Max!

### DISCUSSION THREAD

**Discussion by JLChamberlain** [NightscoutFoundation/xDrip#2449] [Q&A] [answered]
Device: Pixel 7 Pro
OS: Android 13 - November Update
XDrip Version: [November 8th Nightly](https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.11.08)

No matter what I do when I set up the alarms it always plays the alarms at max volume! I don't have any of the override options enabled. I don't want my alarms to play at max volume! I want the alarms to be at the same volume that is set on the device.

**Comment by Navid200** [comment, marked as answer]
Would you please post a screenshot of the following page?

Settings &#8722;> Alarms and Alerts &#8722;> Glucose Alerts Settings

**Reply by JLChamberlain** [reply]
Here you go:
![Screenshot_20221110-181735](https://user-images.githubusercontent.com/22307144/201175197-24e76c5b-defa-4177-8a97-baad3a6cc53e.png)
![Screenshot_20221110-181740](https://user-images.githubusercontent.com/22307144/201175199-f911861d-1711-43e5-a951-3c133a233a00.png)

**Reply by Navid200** [reply]
xDrip has a lot of settings.  It may not be very easy to find everything.

The very first setting, in your screenshot, is set to High.
That means all your Glucose level alerts will run at high volume.  
You can change that to medium.
Even better, in my opinion, you can set it to ascending.  

This page gives a little info about volume profiles: https://navid200.github.io/xDrip/docs/Volume-profiles.html
This page explains what ascending volume profile is:  https://navid200.github.io/xDrip/docs/Ascending-volume-profile.html

And this page is about all alerts:  https://navid200.github.io/xDrip/docs/Alerts_page

**Reply by JLChamberlain** [reply]
So are you saying that in XDrip I can't set it to follow my current device's volume level? The alert volume will follow the "Alert Volume Profile"? The options of high (100%), medium (70%), Ascending (volume increase), Vibrate & silent seem constrictive! If it does follow the "Alert Volume Profile" there should at least be an option to set a custom volume level (e.g 50%, 55%, 60%, etc.).

**Reply by Navid200** [reply]
> So are you saying that in XDrip I can't set it to follow my current device's volume level? 

Not for the Glucose level alerts.  The glucose level alerts are forced to use the volume profile.
This is intentional.  We use our phones for everything and we adjust the volume for what we are selectively listening to.  So, we may lower the volume while listening to a low-volume clip in an extremely quiet environment.
Then, if the alert goes off when we are in a very noisy environment , and we don't hear it, that will be a serious problem.
To avoid that, the main alerts ignore the device alert level.

Low levels are intentionally left out of the volume profile, for the same reason.
An alert is supposed to get your attention.   Otherwise, it's not an alert.

I suggest you try the ascending volume profile.

**Reply by Ferz1984** [reply]
Hello...
I had the same problem and the easiest solution I found is to set the volume to medium and change the alert sound to a 3 semi-silent beeps :) ( i have searched notification sounds on google and download the one I like )
